tests/test_schedule_tools.py

```diff
@@ -89,10 +89,8 @@ def test_calc_duration_with_valid_start_and_end_time_should_return_the_duration(
     assert_that(tools.calc_duration("13:00", "14:00")).is_equal_to("1:00:00")
 
 
-def test_calc_duration_with_reveresed_start_and_end_time_should_throw_an_error():
-    assert_that(tools.calc_duration).raises(
-        ValueError
-    ).when_called_with("14:00", "13:00").is_equal_to("end_time should be greater the start_time")
+def test_calc_duration_with_greater_start_time_than_end_time_should_assume_next_day():
+    assert_that(tools.calc_duration("14:00", "13:00")).is_equal_to("11:00:00")
```

Comment:
Can you explain the logic how 11 hours duration is created by 14:00 to 13:00 on the next day?

Reply:
Well, obviously, since a day has 11 hours that should work :P Probably shouldn't commit code at 1AM.
Fixed now